src/components/config/index.ts

```diff
@@ -83,6 +85,18 @@ const handshakeAndValidate = async (config: Config) => {
   }
 }
 
+const writeConfigFile = (flags: any, parsedConfig: Config) => {
+  const file = flags.output || 'monika.json'
+  writeFile(file, JSON.stringify(parsedConfig), 'utf8', function (err) {
+    if (err) {
+      log.error('An error occured while writing har config to File.')
```

Comment:
Please change the message to 

```
Could not write the new configuration to ${file}.
```

Because the new config can also from postman, not only har.

Reply:
updated